Meziantou.FluentAssertionsAnalyzers.Tests/NUnit4ToFluentAssertionsAnalyzerUnitTests.cs

```diff
@@ -847,4 +847,66 @@ public void MyTest()
             }
             """);
     }
+    
+    [Fact]
+    public Task Assert_AreEqual_on_assemblies()
+    {
+        return Assert(
+            """
+            using NUnit.Framework.Legacy;
+            using System.Reflection;
+
+            class Test
+            {
+                public void MyTest(Assembly assembly1, Assembly assembly2)
+                {
+                    [|ClassicAssert.AreEqual(assembly1, assembly2)|];
+                }
+            }
+            """,
+            """
+            using System.Reflection;
+            using FluentAssertions;
+            using NUnit.Framework.Legacy;
+
+            class Test
+            {
+                public void MyTest(Assembly assembly1, Assembly assembly2)
+                {
+                    assembly2.Should().BeSameAs(assembly1);
```

Comment:
Are you sure of this change? It looks like you change the semantic here.

- `ClassicAssert.AreSame()` => `Should().BeSameAs()`
- `AreEqual` => `Should().Be()`

Reply:
I haven't dig too much in NUnit, but it seems AreEqual has many comparers based on the actual type: https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Constraints/NUnitEqualityComparer.cs#L34-L55

Reply:
Yeah you're right my change is wrong. The problem is specific to TestAssembly` type.

In Fluent it uses:

```csharp
public class AssemblyAssertions : ReferenceTypeAssertions<Assembly, AssemblyAssertions>
```

And it does not inherit from `ObjectAssertions` that provides the `Be` method.

Reply:
I updated the code to target only the Assembly type, let me know what you think.